3rd-party/class.jetpack-amp-support.php

```diff
@@ -176,7 +176,7 @@ public static function amp_post_template_metadata( $metadata, $post ) {
 			$metadata = self::add_site_icon_to_metadata( $metadata );
 		}
 
-		if ( ! isset( $metadata['image'] ) ) {
+		if ( ! isset( $metadata['image'] ) && isset($post) ) {
```

Comment:
I'd suggest going one step further:
 
```suggestion
		if ( ! isset( $metadata['image'] ) && ! empty( $post ) ) {
```

Reply:
Not really a php developer, just wanted to fix this bug on my website 😄 thanks for showing me around that `empty()` encompasses other cases missed by `isset()`.